docs/user_guide/mitigation.rst

```diff
@@ -74,6 +74,109 @@ Preprocessing
    
 .. currentmodule:: fairlearn.preprocessing
 
+Preprocessing algorithms transform the dataset to mitigate possible unfairness
+present in the data.
+Preprocessing algorithms in Fairlearn follow the :class:`sklearn.base.TransformerMixin`
+class, meaning that they can :code:`fit` to the dataset and :code:`transform` it
+(or :code:`fit_transform` to fit and transform in one go).
+
+.. _correlation_remover:
+
+Correlation Remover
+~~~~~~~~~~~~~~~~~~~
+Sensitive features can be correlated with non-sensitive features in the dataset.
+By applying the :code:`CorrelationRemover`, these correlations are projected away
+whiledetails from the original data are retained as much as possible (as measured
+by the least-squares error). The user can control the level of projection via the
+:code:`alpha` parameter. In mathematical terms, assume we have the original dataset
+:math:`X` which contains a set of sensitive attributes :math:`S` and a set of
+non-sensitive attributes :math:`Z`. The removal of correlation is then
+mathematically described as:
+
+.. math::
+        \min _{\mathbf{z}_{1}, \ldots, \mathbf{z}_{n}} \sum_{i=1}^{n}\left\|\mathbf{z}_{i}
+        -\mathbf{x}_{i}\right\|^{2} \\
+        \text{subject to} \\
+        \frac{1}{n} \sum_{i=1}^{n} \mathbf{z}_{i}\left(\mathbf{s}_{i}-\overline{\mathbf{s}}
+        \right)^{T}=\mathbf{0}
+
+The solution to this problem is found by centering sensitive features, fitting a
+linear regression model to the non-sensitive features and reporting the residual.
+The columns in :math:`S` will be dropped from the dataset :math:`X` but the
+amount of correlation that is removed can be controlled using the
+:code:`alpha` parameter. This is described as follows:
+
+.. math::
+        X_{\text{tfm}} = \alpha X_{\text{filtered}} + (1-\alpha) X_{\text{orig}}
+
+Note that the lack of correlation does not imply anything about statistical dependence.
```

Comment:
Maybe add an additional line to explain that correlation measures linear relationships and that non-linear dependencies can still exist in the dataset.